src/main/java/movie/storage/service/impl/CinemaHallServiceImpl.java

```diff
@@ -2,15 +2,19 @@
 
 import java.util.List;
 import movie.storage.dao.CinemaHallDao;
-import movie.storage.lib.Inject;
-import movie.storage.lib.Service;
 import movie.storage.model.CinemaHall;
 import movie.storage.service.CinemaHallService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
 
 @Service
 public class CinemaHallServiceImpl implements CinemaHallService {
-    @Inject
-    private CinemaHallDao cinemaHallDao;
+    private final CinemaHallDao cinemaHallDao;
+
+    @Autowired
```

Comment:
should we use Autowired above constructor or not?
let's create our project in one style

Reply:
I used above all service constructors in the same style.